staff/fernando-costa/bash/README.md

```diff
@@ -0,0 +1,70 @@
+# bash
+Command-line interface commands typical in BASH.
+
+## pwd
+pwd= path work directory => which folder we are now
+```bash
+$ pwd
+```
+
+## ls
+all the folders
+```bash
+$ ls
+```
+
+## ls -l
+all folders we have, but with more info
+```bash
+$ ls -l
+```
+
+## touch    
+it creats a file i.e. hello.world
+```bash
+$ touch
+```
+
+## ls -a
+hidden folders
+```bash
+$ ls -a
+```
+
+## mkdir (make directory)
+it creats a new directory
+```bash
+$ mkdir
```

Comment:
The name of the folder that you want to create should appear next to the "mkdir" comand 😗

Reply:
It's done now :)